### ISSUE TITLE
Reply quotation format bug.

### ISSUE THREAD

**Issue by leftside2** [HelloZeroNet/ZeroMe#114]
Original text: 
> hello
>> hi
> where are u from?
>> I'm from Mars.

Converted result: (after hit Enter)
> hello
>> hi
>> where are u from?
>> I'm from Mars.

To bypass the bug:
> hello
>> hi
.
> where are u from?
>> I'm from Mars.

**Comment by krixano** [comment]
I don't think this is a bug. New lines always go into the blockquote unless there's two new lines. For example:
```
> test
still inside blockquote
```
will have that second line still inside the blockquote. Whereas:
```
> test

not inside blockquote
```
Putting two new lines will make the line not inside the blockquote.

In your case, you only have one new line after the first *inner* blockquote, so all of the lines are inside it.

**Comment by leftside2** [comment]
@krixano : So do you mean that if someone think it's ridiculous he has to ask CommonMark staff? Any room for ZeroMe developer has something to do with this matter?

**Comment by krixano** [comment]
@leftside2 I was merely saying it's not a bug but that it's intended as per the standard. Nofish might be able to fix this, but chances are he's using a library for this - so he'll have to change that library.

**Comment by leftside2** [comment]
@krixano : I see. :) One question. Plz answer. In ZeroMe, In the new post box, by pressing SHIFT-ENTER you can feed new line without submitting the post. But in the comment box under the post this key combination doesn't work. In my case, I click [reply] button on the other comments to feed the new line and erase the ID part to make use of the new line. How do u do to do this?

**Comment by HelloZeroNet** [comment]
tab + enter should work. (maybe we should remove the comment sending by enter to make it consistent)

**Comment by leftside2** [comment]
@HelloZeroNet : Thanks! Why don't u put a [?] icon beneath the test editing box?

**Comment by leftside2** [comment]
@HelloZeroNet : tab, enter doesn't work. :(

**Comment by HelloZeroNet** [comment]
I just tried and it's works, to posting on ZeroMe

**Comment by leftside2** [comment]
@HelloZeroNet : I mean the reply edit box, not new post. :)
